packages/components/src/components/files/FileAttachmentContainer.tsx

```diff
@@ -25,6 +25,7 @@ interface FileAttachmentContainerProps {
     allowDirectories: boolean
     handleChange?: any
     handleRemoval?: any
+    index?: any
```

Comment:
Should this type be "number"? or what is an example of what a usage will pass in as this property?

Reply:
Fixed